site/viamguides/src/pomodoro-bot/pomodoro-bot.md

```diff
@@ -0,0 +1,439 @@
+
+### Hardware Setup
+
+* **Components:**
+    * TEMT6000 Ambient Light Sensor
+      ![TEMT 6000](assets/temt6000.png)
+    * ADS1115 Analog-to-Digital Converter (ADC)
+      ![ADS1115 ADC](assets/ads1115.png)
+
+* **Connections:**
+    * **TEMT6000 to ADS1115:**
+        * Connect the analog output pin of the TEMT6000 to one of the analog input channels (e.g., A0) on the ADS1115.
+        * Connect the VCC pin of the TEMT6000 to the 3.3V or 5V power supply.
+        * Connect the Ground pin of the TEMT6000 to the Ground pin on the ADS1115.
+
+    * **ADS1115 to Raspberry Pi:**
+        * Connect the SCL pin of the ADS1115 to the SCL pin on the Raspberry Pi.
+        * Connect the SDA pin of the ADS1115 to the SDA pin on the Raspberry Pi.
+        * Connect the VCC pin of the ADS1115 to the 3.3V pin on the Raspberry Pi.
+        * Connect the Ground pin of the ADS1115 to a Ground pin on the Raspberry Pi.
+
+### Software Integration
+
+* **Add the TEMT6000 Sensor in Viam App:**
+    1. Open the Viam application.
+    2. Navigate to the **CONFIGURE** tab.
+    3. Add the TEMT6000 sensor as a new component, specifying its ADC channel.
+    ![addTEMT6000](assets/addTEMT6000.png)
+    ![TEMT6000 Configuration](assets/temt6000Config.png)
+
+* **Testing the Sensor:**
+    1. Switch to the **Control** tab in the Viam app.
+    2. Observe the live data feed from the TEMT6000 sensor.
+       ![TEMT6000 Test](assets/testTEMT6000.png)
+    3. Adjust the lighting around the sensor and verify that the readings update in real time.
+
+
+##  Integrate a 4-Inch HDMI Capacitive Touch Display
+
+Duration: 10
+
+Adding a display to your Pomodoro Bot elevates it from a functional productivity tool to an interactive and engaging companion. With a screen, your bot can visually communicate, provide intuitive feedback, and motivate you with friendly reminders or fun animations. 
+
+![Display](assets/display.jpg)
+
+**Features**
+
+* 4-inch IPS screen, hardware resolution is 720 × 720.
+* 5-point capacitive touch, toughened glass panel, hardness up to 6H.
+* When used with Raspberry Pi, it supports Raspberry Pi OS / Ubuntu / Kali and Retropie.
+* Ubuntu is supported when used with Jetson Nano.
+* When used as a computer monitor, it supports Windows 11/10/8.1/8/7.
+* Onboard dual touch circuit, optional USB Type-C or I2C touch, it has more application scenarios.
+* With 3.5mm audio and speaker interface, it supports HDMI audio output.
+
+### Hardware Connection
+
+1. Set the touch switch on the back of the screen to "I2C".
+2. Fix the Raspberry Pi to the screen through the copper posts, and pay attention to aligning the position of the ejector pins.
+3. Connect the HDMI port of the Raspberry Pi to the LCD.
+4. Connect USB-C on Display to USB port on Raspberry Pi to enable touch interactions.
```

Comment:
Are they connecting the USB-C on the display to one of the USB-A ports specifically? Does it matter if it's a USB 3.0 or 2.0 port?